### DISCUSSION TITLE
IOTA DID integration

### DISCUSSION THREAD

**Discussion by JamesonBR** [walt-id/.github#28] [Help] [answered]
Hello, I'm running a iota private tangle and would like to know how to create a did directly on my server?

**Comment by severinstampler** [comment, marked as answer]
Hi @JamesonBR 
We don't currently support private tangles. Though I think it wouldn't be hard to implement.
I've created an issue to keep it in mind: https://github.com/walt-id/waltid-ssikit/issues/195
br 
Severin
